src/main/java/yahav/photomosaic/PhotoMosaicApplication.java

```diff
@@ -52,18 +50,20 @@ public void start(Stage primaryStage) throws Exception {
                 Color color = pixelReader.getColor(x,y);
                 System.out.print("section " + x + ", ");
                 System.out.println(y);
-                double red = ((color.getRed() * 255)/3);
-                double green = ((color.getGreen() * 255)/3);
-                double blue = ((color.getBlue() * 255)/3);
+                double red = ((color.getRed())/3);
```

Comment:
In order for there not to be an illegal argument I had to remove the 255 part. I test ran this loop to see if it creates a pixelated version and it does but with spacing in betwee each box.